discovery-infra/utils.py

```diff
@@ -356,6 +359,10 @@ def is_cluster_in_status(client, cluster_id, statuses):
         log.exception("Failed to get cluster %s info", cluster_id)
 
 
+def get_random_name(length=8):
```

Comment:
added to the wrong utils file

This one is no longer with us

Reply:
Fixed